apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+  /batch/list:
+    post:
+      description: >-
+        This API is associated with fetching and listing existing batches
+          - The format of the resource URL is '{{host}}/course/v1/batch/search'
+          - The **_/batch/list_** endpoint executes "Lists the existing batches" request to fetch the existing batches.
+          - It is a must to provide values for required parameters 
+      summary: Lists the existing batches
+      tags:
+      - Course Batch APIs 
+      operationId: '{{host}}/course/v1/batch/list'
+      produces:
+      - application/json
+      parameters:
+      - in: query
+        name: orgdetails
+        description: Extra data can be passed in query param with above mention key and comma separated value:Example ,?orgdetails=email,name . Sunbird will take this query param and pass to content service as it is, and content service knows what need to be done.
+        type: string
+        required: false 
+      - name: Body
+        in: body
+        required: true
+        description: >-
+            The body refers to the format of the request.
+              - The body contains metadata about the batches to be listed
+        schema:
+         allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/CourseBatchSearchReq'
+      
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp identifying when the **List the existing batches** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID which identifies the request incase the same API is executed multiple times.'
+      - name:  x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The alphanumeric string for accessing the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'

```

Comment:
Authorization is mandatory for using an API. Request the administrator for the access key. Enter the key received through email